changelog.md

```diff
@@ -1,5 +1,35 @@
 ## Changes from 1.5.x to 1.6.0
 
+Recommended Mesos version is 1.5.0.
+
+### New versioning strategy
+As of 1.6 version, Marathon switched to the new versioning strategy. We still follow [SemVer](https://semver.org/) but we needed to have a unique version number for every master build. For that reason, first 1.6 public release is 1.6.322. It does not mean there were 321 releases prior to this one that we did not share with the community, the number actually is number of commits on the master branch since the last minor release (so 1.5). As for all future releases, the fix version will be increasing but don't expect the numbers to be consequential.
+
+### New features
+
+#### Performance improvements
+As a part of performance improvement initiative we did a 10X improvement on the performance of the v2/groups endpoint [#5973](https://github.com/mesosphere/marathon/pull/5973)
+
+#### Maintenance mode support
+As of version 1.6, Marathon has simple built-in support for [Mesos Maintenance Primitives](http://mesos.apache.org/documentation/latest/maintenance/). For more informations please see our [docs](https://mesosphere.github.io/marathon/docs/maintenance-mode.html).
```

Comment:
`https://mesosphere.github.io/marathon/docs/maintenance-mode.html` gives a 404.

Reply:
this will work after we release docs...